radixdlt-java/src/main/java/com/radixdlt/client/application/translate/PowFeeMapper.java

```diff
@@ -24,21 +25,21 @@
 public class PowFeeMapper implements FeeMapper {
 	private static final int LEADING = 16;
 
-	private final Function<List<ParticleGroup>, RadixHash> hasher;
+	private final Function<Atom, RadixHash> hasher;
 	private final ProofOfWorkBuilder powBuilder;
 
-	public PowFeeMapper(Function<List<ParticleGroup>, RadixHash> hasher, ProofOfWorkBuilder powBuilder) {
+	public PowFeeMapper(Function<Atom, RadixHash> hasher, ProofOfWorkBuilder powBuilder) {
```

Comment:
This interface change needs some changes in PowFeeMapperTest that haven't been made, and prevent the unit tests from compiling / running at the moment.  Probably a good habit to get into running `gradle check` before pushing.